cmd/tether/utils_linux.go

```diff
@@ -90,6 +91,23 @@ func (t *osopsLinux) SetHostname(hostname string) error {
 	return nil
 }
 
+func pciToMAC(pciAddr string) (string, error) {
+	addrPath := "/sys/bus/pci/devices/" + pciAddr + "/net/*/address"
+	f, err := os.Open(addrPath)
+	if err != nil {
+		return "", fmt.Errorf("unable to open addrPath %s: %s", addrPath, err)
+	}
+	defer f.Close()
+
+	buf := make([]byte, 18)
```

Comment:
Where is that 18 value coming from?I don't think we need precise buffering here, you can just use ioutil.ReadAll here.